test/integration/tests/button/renderOrder.js

```diff
@@ -0,0 +1,62 @@
+/* @flow */
+/* eslint max-lines: 0 */
```

Comment:
Is this esline max-lines override needed? I wouldn't expect it to be since this file only 62 lines long.
```suggestion
```

Reply:
You're right, thanks for the catch!